.github/workflows/link-docs-preview.yml

```diff
@@ -17,21 +17,16 @@ jobs:
       contents: read
       pull-requests: write
     steps:
-      - name: Quick debug
+      - name: Get hash for branch
+        id: hash
         run: |
-          echo ${{ github.ref_name }}
-          echo ${{ toJSON(github.event) }}
-          md5 -qs "fooey" || echo 'no md5 here'
-          echo 'fooey' | md5sum || echo 'no md5sum here'
-      # - uses: pplanel/hash-calculator-action@v1.3.1
-      #   id: hash
-      #   with:
-      #     input: ${{ github.evenet.pull_request.head.ref }}
-      #     method: MD5
-      # - uses: peter-evans/create-or-update-comment@v3
-      #   with:
-      #     token: ${{ secrets.DETERMINED_TOKEN }}
-      #     issue-number: ${{ github.event.number }}
-      #     body: |
-      #       Docsite preview being generated for this PR.
-      #       You can (eventually) find the generated docsite [here](https://determined-ai-docs.s3.us-west-2.amazonaws.com/${{ steps.hash.outputs.digest }}/index.html).
+          echo "branch: ${{ github.event.pull_request.head.ref }}"
```

Comment:
I think this is vulnerable to [script injection](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#example-of-a-script-injection-attack), which I just came across in the GitHub docs. Looks like you should use an environment variable instead.

Reply:
Solid catch, thanks a bunch. I'll switch to an env var rn